sql3/sql_complex_test.go

```diff
@@ -2978,3 +2978,62 @@ func TestPlanner_BulkInsertParquet(t *testing.T) {
 		assert.Equal(t, row[1], row[2])
 	})
 }
+
+func TestPlanner_BulkInsert_FP1916(t *testing.T) {
+	c := test.MustRunCluster(t, 3)
+	defer c.Close()
+	node := c.GetNode(0).Server
+	_, _, _, err := sql_test.MustQueryRows(t, node, `create table greg-test (
```

Comment:
I have two suggestions for this:

1. Drop a bunch of these fields because we're not using them.
2. Make the number being converted more obviously defined in terms of the limit we're testing against, or possibly the range of int. Say, use `1<<62`.
3. Maybe also test at least one number that's out of even that expanded range and confirm that we actually handle it... however we expect to handle it?

... I recognize that's not actually two but I'm just gonna run with it.